storybook/src/data/lists.data.js

```diff
@@ -1,3344 +1,109 @@
-export const LIST_EMPTY = {
-  properties: {
-    data: [],
-  },
-};
+import DocumentBuilder from './documents.data';
+
+import image01 from '../img/image01.jpg';
```

Comment:
we could just use one of the numerous image placeholders CDN (ie https://placeholder.com/ )

Reply:
BTW why are we importing images ?! Will webpack inline these?

Reply:
Since we are trying to have some control of the visuals (so the final product looks great), I suggest keeping these images for now.